### PR TITLE
[SERVICES] - [INTEGRATION] - milestone: adding Model , index , contro…

### PR DESCRIPTION
**Pull request by RicardoJ**
…llers

### CODE REVIEW

File: services/integration/src/controllers/constitution/milestone.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const { projectId } = req;
+
+  Milstone.insertMany(milestone.map(m => ({ ...m, projectId })));
+
+  res.send({ message: 'Milestones Created successfully' });
+});
+
+const find = asyncHandler(async (req, res) => {
+  const milestone = await Milstone.find({});
+  console.log(milestone);
```
Comment (juribee) [inline services/integration/src/controllers/constitution/milestone.js]: Remove this console.log

File: services/integration/src/models/constitution/milestone.model.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const milestoneSchema = new Schema({
+  projectId: {
+    type: String,
+    required: [true, 'ProjectId is a required field']
+  },
+  name: String,
+  description: String,
+  deliverable: String,
+  date: String
```
Comment (juribee) [inline services/integration/src/models/constitution/milestone.model.js]: This must be date type

File: services/integration/src/routes/index.js [hunk 3]
```diff
@@ -23,4 +24,13 @@ Router.use(
   ObjectivesRouter
 );
```
Comment (juribee) [inline services/integration/src/routes/index.js]: You are breaking the patter here.  Remove this space line

File: services/integration/src/models/constitution/milestone.model.js (hunk 2 above)
Comment (RicardoJ) [reply services/integration/src/models/constitution/milestone.model.js]: in trello you said , date : string

File: services/integration/src/routes/index.js (hunk 3 above)
Comment (RicardoJ) [reply services/integration/src/routes/index.js]: are you kidding?

File: services/integration/src/controllers/constitution/milestone.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  const { projectId } = req;
+
+  Milstone.insertMany(milestone.map(m => ({ ...m, projectId })));
+
+  res.send({ message: 'Milestones Created successfully' });
+});
+
+const find = asyncHandler(async (req, res) => {
+  const milestone = await Milstone.find({});
+
```
Comment (juribee) [inline services/integration/src/controllers/constitution/milestone.js]: what about those empty lines?

**Pull request merged**